tests/test_helper/bats/cluster.bats

```diff
@@ -208,4 +211,23 @@ cluster_test_db_connection_string() {
     done
 }
 
+# _test_southbound_propagation
```

Comment:
nit: typo in the name

Reply:
The function is called `cluster_test_southbound_propagation`, you are missing the `cluster` part in the docstring. I see that similar "typo" is in the function above, but the other documented function in this file has the full name.
I'm not sure what's the reasoning behind leaving the "cluster" part out of the name, but I personally prefer having the full name in the docstring because it makes it easy to search for the function definition with `grep -iRn "# <function_name>" tests/`

Reply:
reasonable, saw it in the function above and assumed it was practice, should i change it in the function above, maybe in a seperate commit?